### DISCUSSION TITLE
Hello Everyone!

### DISCUSSION THREAD

**Discussion by Praful932** [OpenMined/courses#12] [Introduce Yourself]
I am a third year Undergraduate pursuing Information Technology Engineering. Aim to contribute to this wonderful community with their awesome libraries. Super Excited to be here! 🎯

**Comment by yemikifouly** [comment]
And we are super excited that you are here! 🙂 And that is one awesome goal that you have there! At OpenMined we are always looking for contributors! Also, taking the courses is definitely a great way to prepare to join one of our teams! So you are already well on your way to reaching your goal. 🙂💪
